abilitybot/startScenarioExample.js

```diff
@@ -0,0 +1,65 @@
+import { CharInfo } from "../bossroom";
+
+export function configureBot(rg) {
+    rg.automatedTestMode = true;
```

Comment:
I wonder if we should assume this value to be true if they define `startScenario`.  I know that is artificial and somewhat limiting, but it also seems to align with how we're defining that `startScenario` should be utilized.

Reply:
Just to spark my memory here, what is the implementation difference between configureBot and startScenario?

Reply:
configureBot is now purely for bot configuration. It cannot run any bot logic.
startScenario is the "new" configureBot